win32/config.gc

```diff
@@ -307,8 +307,8 @@ d_ilogbl='define'
 d_inc_version_list='undef'
 d_index='undef'
 d_inetaton='undef'
-d_inetntop='undef'
-d_inetpton='undef'
+d_inetntop='define'
```

Comment:
Won't that cause issues on older versions? Or did those already have these functions but just needed to include winsock2.h

Reply:
I wondered about that for the first time as I was creating that PR ... but I didn't really follow it through because I had found no issue with it in the testing I had done. (This morning I again ran a build of patched blead using gcc-8.3.0, runtime 6.0.0, including 'make test' ... just to be sure.)

The thing is that the current blead version of Socket.xs unconditionally provides inet_ntop() and inet_pton() for win32.
It then unconditionally defines both HAS_INETNTOP and HAS_INETPTON for win32 (at lines 149 and 150).
Aside: I can't actually find any C/XS code that looks for or utilises those 2 definitions. They seem to be purposeless.

I don't think that the contents of config.gc and config_H.gc (wrt these symbols) has any influence on the functionality of perl, irrespective of gcc version and mingw runtime version.
But the absence of these config.gc definitions might conceivably result in tests being unnecessarily skipped.

So these changes to config.gc and config_H.gc are not really part of the fix for this issue - and sorry for not having been aware enough to mention that.
They are, IMO. changes that could have been made (and should have been made) a while back.
It seems like an opportune moment to introduce them now.

Cheers,
Rob